charts/ks-devops/crds/devops.kubesphere.io_pipelineruns.yaml

```diff
@@ -0,0 +1,137 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Other CRDs using the version `v1` instead of `v1beta1`. See also [here](https://github.com/kubesphere/ks-devops/blob/63c99e17a2a744c8321a4e71b05889dee151e51b/config/crd/bases/devops.kubesphere.io_pipelines.yaml#L3).

Reply:
So I will keep this deprecated version in this PR until #189 is resolved.

Reply:
I don't think #189 blocked you.